docs/en/interfaces/formats/Avro/_snippets/common-format-settings-avro.md

```diff
@@ -0,0 +1,8 @@
+| Setting                                     | Description                                                                                         | Default |
+|---------------------------------------------|-----------------------------------------------------------------------------------------------------|---------|
+| `input_format_avro_allow_missing_fields`    | For Avro/AvroConfluent format: when field is not found in schema use default value instead of error | `0`     |
+| `input_format_avro_null_as_default`         | For Avro/AvroConfluent format: insert default in case of null and non Nullable column	             |   `0`   |
+| `output_format_avro_codec`                  | Compression codec used for output. Possible values: 'null', 'deflate', 'snappy', 'zstd'.            |         |
+| `output_format_avro_sync_interval`          | Sync interval in bytes.	                                                                         | `16384` |
```

Comment:
@Avogar just want to confirm that the last three settings in this table are applicable to both Avro and AvroConfluent or only to Avro?

Reply:
We don't support `AvroConfluent` output format, only input